app.js

```diff
@@ -9,25 +9,25 @@ app.get('/api', getEndpoints);
 
 app.get('/api/articles/:article_id', getArticleById);
 
-// app.use((req, res, next) => {
-//     const error = new Error("Not Found");
-//     error.status = 404;
-//     next(error);
-// });
+app.get('/api/articles/:article_id/comments', getCommentsByArticleId);
+
+app.all("/*", (req, res, next) => {
+  res.status(404).send({ msg: "Path not found" });
+});
 
 app.use((err, req, res, next) => {
     console.log(err);
```

Comment:
I would recommend moving this console log on line `19` to be inside the _else_ block on line `26 - 28` 
The reason I say this is that currently the log will fire every time you get an error, which will fill your terminal with logs when running the test suite. You only really want that log to fire when it's an unhandled or unexpected error, so it's right at home in the `500` section

Reply:
done